data/heliarch intro.txt

```diff
@@ -0,0 +1,1568 @@
+			`	"Now, Captain, to start looking for something suspicious, where would you?" He asks, showing you some sort of identification card, and explaining that it can open the doors here, but you really can't make out any differences between any of the many warehouses here. One or two have a bit of rust on the doors, and some of them appear to have some rain dripping it's way inside, but aside from small details you can't make out any striking characteristic of the Arachi warehouses.`
+			choice
+				`	(Check the warehouses with the rusty doors.)`
+					goto rustybeetle
+				`	(Check the warehouses with infiltration.)`
+					goto reactor
+				`	(Go further out and check the trees and bushes.)`
+					goto trees
+			label rustybeetle
+			`	Kirrie uses his card to open the door, and continues to explain that it is an access card granted to Heliarch agents, in case they need to do something like this, or just identify themselves as one.`
+			`	Sadly the warehouse turns out to not contain anything significant. It's a storage for what Kirrie says are toy parts, just mountains of plastic stacked together.`
+			`	You check a few more like this, but none turn out to have anything. The rain is starting to get heavier.`
+			choice
+				`	(Check the warehouses with infiltration now.)`
+					goto reactor
+				`	(Go further out and check the trees and bushes.)`
+					goto trees2
+				`	(Leave the rest of this to Kirrie.)`
+					goto whiteflagbeetle
+			label whiteflagbeetle
+			`	You apologize and tell Kirrie that you don't know what to do next, but while he does agree to do it himself, he never stops asking for your opinion on his choices.`
+			`	You spend a few more hours checking the warehouses, until the rain and the wind get too dangerous for you to be moving from warehouse to warehouse, so you take shelter in one of them until the weather settles. Once it does, near nightfall, he gets notified by Iplora that she and Tugroob are already back at the ship, so you two decide to head back too.`
+			`	It seems Tugroob found some evidence of some smuggled weapons passing through here, but nothing major. They get back in their bunks, saying you may depart now.`
+				accept
+			label reactor
+			apply
+				"resistance evidence" ++
+			`	Kirrie tries to open the door, but can't get his card to work.`
+			`	"All the doors to these warehouses, my card should open. Damaged, the reader doesn't seem to be," he says. He decides that he must get in, and grabs a small sidearm, busting the door open after some shots.`
+			`	There is nothing inside, at first glance, but as he crawls all over the place with furious speed, he puts his head to a large metal panel on the opposite end of the room, and shoots it open too, revealing a hidden staircase leading down.`
```

Comment:
`	Aat first glance there is nothing inside, but as he crawls all over the place with furious speed, he puts his head to a large metal panel on the opposite end of the room, before shooting that too, revealing a hidden staircase leading underground.`